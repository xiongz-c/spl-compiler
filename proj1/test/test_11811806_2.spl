struct struct1 {
    int attr = 10;
}s1, s2[3]

struct struct2 {
    int arr[10];
s1;

struct struct3 {
    int num = 1;
}s1[10;

int 00_main_1() {
    int a = ;
    return 0;
}