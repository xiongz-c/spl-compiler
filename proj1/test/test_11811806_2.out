Error type B at Line 3: Missing semicolon';'
Error type B at Line 6: Missing '}'
Error type B at Line 11: Missing ']'
Error type A at Line 13: unknown lexeme 00_main_1
Error type B at Line 14: Definition not match rules
